### PR TITLE
Running black

### PR DESCRIPTION
**Pull request by ajyl**
Ran [Black](https://black.readthedocs.io/en/stable/index.html) to clean up/standardize the code. Black is an automated styling tool that tries its best to adhere to PEP 8's guidelines. See also: [Code Style](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-black-code-style)

All I did was run
```
black ./
```
from the root directory. 
This PR was made from #17 , but once #17 gets in I don't think there should be any merge conflicts...

### CODE REVIEW

File: potato/create_task_cli.py [hunk 1]
```diff
@@ -56,78 +56,112 @@ def get_initial_config():
...
-    print('the intructions, add fancier features, or even change the nature of the')
-    print('annotation. This process will only take a few minutes.')
-    print('')
-    print('If you don\'t yet know the answer to any of these questions you can leave')
-    print('the answer blank and edit your .yaml file later to fill in these details.')
-
-
-    print('')
-    print('First, some preliminary questions about how you want the server setup:')
-
```
Comment (davidjurgens) [inline potato/create_task_cli.py:82]: Any chance to revert this part? Some of the Black changes are kind of ... visually annoying. I'm on board with most of PEP-8 but not everything :)

File: potato/create_task_cli.py [hunk 2]
```diff
@@ -136,56 +170,73 @@ def create_task_cli():
...
+            if atype == "likert":
+                size = str(input("How many items are on the likert scale?\n"))
             else:
-                size = str(input('How many items to dislay at once for Best-Worst Scaling?\n'))
+                size = str(
+                    input(
+                        "How many items to dislay at once for Best-Worst Scaling?\n"
+                    )
+                )
+
```
Comment (davidjurgens) [inline potato/create_task_cli.py:196]: I really don't understand why Black is making this particular kind of change. Can we turn off certain flags for style differences?

**Review by davidjurgens — CHANGES_REQUESTED** [review]
Overall it looks good  though there's some (for me) very ugly changes where a single line that's not too long gets turned into 5+ lines. I wonder if we could turn off those changes? That might be a pain though given this conversion already :|

File: potato/create_task_cli.py (hunk 2 above)
Comment (ajyl) [reply potato/create_task_cli.py:157]: Done -- simply ran `black --line-length 100`. I think PEP 8 recommends line length of 80 characters, which is what Black was using. Anything longer than 100 characters should probably be broken up...

File: potato/create_task_cli.py (hunk 1 above)
Comment (ajyl) [reply potato/create_task_cli.py]: Done -- simply ran `black --line-length 100 ./`!

**Review by davidjurgens — APPROVED** [review]
Looks good to me! The re-run black stuff looks more vertically compressed which is great for readability (imo)

**Pull request merged**